src/medal_app/events.cljs

```diff
@@ -1,10 +1,8 @@
 (ns medal-app.events
-  (:require
-   [re-frame.core :as re-frame]
-   [day8.re-frame.http-fx]
-   [ajax.core :as ajax]
-   [medal-app.db :as db]
-   ))
+  (:require [re-frame.core :as re-frame]
+            [day8.re-frame.http-fx]
+            [ajax.core :as ajax]
+            [medal-app.db :as db]))
```

Comment:
Just a style suggestion. This is how we usually format `:require` statements in rcp_reuters-connect-website